images/scripts/opensuse-tumbleweed.setup

```diff
@@ -112,6 +112,9 @@ wireguard-tools \
 mkdir -p /etc/NetworkManager/conf.d/
 printf '[main]\nno-auto-default=*\n' > /etc/NetworkManager/conf.d/noauto.conf
 
+# use unpredictable network interface names
+ln -s /dev/null /etc/systemd/network/99-default.link
```

Comment:
This is another case of "please test the real-world default OS, not some hack". We fixed our projects to get along without hardcoded ethernet names a while ago, this really shouldn't be necessary.